CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0 (2015-09-02)
```

Comment:
Sounds good to me. I did it: https://github.com/yola/yolapy/commit/84ca62b3631355f51c12283f169fce04ea293c4b